openshiftcontrolplane/v1/types.go

```diff
@@ -312,6 +312,11 @@ type DockerPullSecretControllerConfig struct {
 	// registry. The value must be in "hostname[:port]" format.  Docker pull secrets
 	// will be generated for this registry.
 	InternalRegistryHostname string `json:"internalRegistryHostname"`
+
+	// serviceAccountSigningPublicKeyFile specifies the path to a file that contains the current public key of the
+	// service account token issuer. The issuer signs issued ID tokens with this private key.
+	// +optional
+	ServiceAccountSigningPublicKeyFile string `json:"serviceAccountSigningPublicKeyFile,omitempty"`
```

Comment:
What happens when this is omitted?
Does this interact with any other fields nearby?
Is there a format for this? Any characters not allowed?
How long can this be? Infinitely?

I would expect to see each of the above documented in the godoc, and where appropriate validated

Reply:
@JoelSpeed I've updated the doc with the omission info, but as to the questions related to validation, I do not think any further validation is appropriate for a field meant to contain a file name. Beyond, the complexity of coming up with a format that could encompass every valid file name, even a "valid" file name can be wrong if the file does not exist. I see no other validation of file name value fields in this repo.

Reply:
We should at least impose a maximal length to the string, `4096` I think would be reasonable, it's the maximum path length on most Unix based systems